spec/queries/get_application_progress_data_by_course_spec.rb

```diff
@@ -50,6 +49,14 @@
       expect(accredited_provider_courses.find { |c| c.status == 'pending_conditions' }.count).to eq(5)
     end
 
+    it 'returns the status for current course associations with applications' do
+      create(:application_choice, status: :withdrawn, course_option: previous_course_option, current_course_option: course_option)
```

Comment:
could we test for statuses we will be surfacing in the report as we don't surface the withdrawn state?

Reply:
:+1: ah sure yes, that was an arbitrary choice